api/hypercore/hypertable_columnstore_settings.md

```diff
@@ -0,0 +1,62 @@
+---
+api_name: timescaledb_information.hypertable_columnstore_settings
+excerpt: Get information about columnstore settings for all hypertables
+topics: [hypercore, information, columnstore, hypertable]
+keywords: [columnstore, hypercore, hypertable, information]
+tags: [hypertable columnstore, columnstore settings]
+api:
+  license: community
+  type: view
+---
+import Since2180 from "versionContent/_partials/_since_2_18_0.mdx";
+
+# timescaledb_information.hypertable_columnstore_settings 
+
+Retrieve information about the settings for all hypertables in the columnstore.
+
+<Since2180 />
+
+## Samples
+
+To retrieve information about settings:
+
+- **Show columnstore settings for all hypertables**:
+
+   ```sql 
+   SELECT * FROM timescaledb_information.hypertable_columnstore_settings'
+   ```
+  Returns:
+   ```sql  
+   hypertable               | measurements
+   segmentby                | 
+   orderby                  | "time" DESC
+   compress_interval_length | 
+   ```
+
+- **Retrieve columnstore settings for a specific hypertable**:
+
+   ```sql
+   SELECT * FROM timescaledb_information.hypertable_columnstore_settings WHERE hypertable::TEXT LIKE 'metrics';
+   ```
+  Returns:
```

Comment:
no longer valid as of v2.18.1. See my PR https://github.com/timescale/docs/pull/3837

Reply:
I've just checked and both work. As this is the api ref for hypertable_columnstore_settings, I'll leave this example as is.